uint16 n_bins
float64 range
float64 vos
uint16 left_limit
uint16 right_limit
uint8 step_angle_size
float64 gain
bool debug_mode
---
bool result
